boxoffice/models/invoice.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+from decimal import Decimal
+from boxoffice.models import db, BaseMixin
+from baseframe import __
+from coaster.utils import LabeledEnum
+
+__all__ = ['Invoice', 'InvoiceLineItem']
+
+
+class Invoice(BaseMixin, db.Model):
+    __tablename__ = 'invoice'
+    __uuid_primary_key__ = True
+    __table_args__ = (db.UniqueConstraint('organization_id', 'invoice_no'),)
+
+    invoicee_name = db.Column(db.Unicode(255), nullable=True)
+    invoicee_email = db.Column(db.Unicode(254), nullable=True)
+    invoice_no = db.Column(db.Unicode(32), nullable=True)
+    invoiced_at = db.Column(db.DateTime, nullable=True)
+    street_address = db.Column(db.Unicode(255), nullable=True)
+    city = db.Column(db.Unicode(255), nullable=True)
+    state = db.Column(db.Unicode(255), nullable=True)
+    country = db.Column(db.Unicode(255), nullable=True)
+    postcode = db.Column(db.Unicode(8), nullable=True)
+    # GSTIN in the case of India
+    taxid = db.Column(db.Unicode(255), nullable=True)
+    # India specific: this is the state short code. Eg: KA for Karnataka
+    place_of_supply = db.Column(db.Unicode(3), nullable=True)
```

Comment:
Use ISO 3166-1 and 3166-2 codes for both country and state.

Also note: India has three ways to represent state codes. [ISO-3166-2:IN](https://en.wikipedia.org/wiki/ISO_3166-2:IN) defines ISO state codes, but some states have different codes for license plates. Uttarakhand (formerly Uttaranchal) is variously `UT`, `UK`, `UA` and `UL`. The tax codes, which are numeric, are a [different series](http://www.caa-ahm.org/Pdf/Legal/Legal-35.pdf).